src/main/java/teammates/client/scripts/RepairTeamNameInStudentAttributesAndResponses.java

```diff
@@ -0,0 +1,152 @@
+package teammates.client.scripts;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.logging.Level;
+
+import javax.jdo.PersistenceManager;
+
+import teammates.client.remoteapi.RemoteApiClient;
+import teammates.common.datatransfer.FeedbackResponseAttributes;
+import teammates.common.datatransfer.StudentAttributes;
+import teammates.common.exception.EntityAlreadyExistsException;
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.common.exception.InvalidParametersException;
+import teammates.common.util.StringHelper;
+import teammates.common.util.Utils;
+import teammates.logic.core.FeedbackResponsesLogic;
+import teammates.logic.core.StudentsLogic;
+import teammates.storage.api.StudentsDb;
+import teammates.storage.datastore.Datastore;
+
+public class RepairTeamNameInStudentAttributesAndResponses extends RemoteApiClient {
+    private final boolean isPreview = true;
+    
+    private StudentsDb studentsDb = new StudentsDb();
+    private StudentsLogic studentsLogic = StudentsLogic.inst();
+    private FeedbackResponsesLogic responsesLogic = new FeedbackResponsesLogic();
+    
+    public static void main(String[] args) throws IOException {
+        RepairTeamNameInStudentAttributesAndResponses migrator = new RepairTeamNameInStudentAttributesAndResponses();
+        migrator.doOperationRemotely();
+    }
+
+    @Override
+    protected void doOperation() {
+        Datastore.initialize();
+        List<StudentAttributes> allStudents = studentsLogic.getAllStudents();
```

Comment:
Then I will keep them as `Student` to check for the extra spaces. That should be okay, right?